### PR TITLE
Fixes routing tab

### LINKED ISSUE #13: Routing tab
When a tab is clicked, the contents of the tab should be displayed. It shouldn't route to a page

### PR DESCRIPTION
**Pull request by Moween**
Display tab content without react-router

### CODE REVIEW

File: src/components/Tabs.jsx [hunk 1]
```diff
@@ -1,40 +1,66 @@
...
+          }
+        }}
+        onChange={(e, index) => setTabIndex(index)}
+        aria-label="Tabs"
+      >
+        <Tab classes={tabItemStyles} disableRipple label="Yesterday" />
+        <Tab classes={tabItemStyles} disableRipple label="Today" />
+        <Tab classes={tabItemStyles} disableRipple label="Tomorrow"  />
+      </Tabs>
+      <TabPanel value={tabIndex} index={0}>
```
Comment (ebukaume) [inline src/components/Tabs.jsx:40]: Does `tabIndex` correspond to `index`? If so can we do `index={tabIndex}`?

File: src/components/Tabs.jsx [hunk 2]
```diff
@@ -1,40 +1,66 @@
...
+      { value === index && (
+        <Box>
+          {children}
+        </Box>
+      )}
+    </>
+  );
+}
 
 export default AppleTabs;
```
Comment (ebukaume) [inline src/components/Tabs.jsx]: Also, end files with a new line, so you don't get this kind of breaks

**Review by ebukaume — COMMENTED** [review]
Looks good but I check it locally and noticed you move the API base URL to `REACT_APP_BASE_URL` env without providing a way for other contributors to know about it.

Consider creating a copy of your `.env` file (eg: .env.sample) with the required envs

File: src/components/Tabs.jsx (hunk 1 above)
Comment (Moween) [reply src/components/Tabs.jsx:40]: No it doesn't corresponds.....

File: src/components/Tabs.jsx (hunk 2 above)
Comment (Moween) [reply src/components/Tabs.jsx]: Okay bros.

File: src/components/Tabs.jsx (hunk 1 above)
Comment (ebukaume) [reply src/components/Tabs.jsx:40]: Ok

**Review by ebukaume — APPROVED** [review]
Approved!

**Comment by Moween** [comment]
Documented `ENV`

**Pull request merged**